### PR TITLE
Add JS tests

### PR DESCRIPTION
**Pull request by silasdavis**

### CODE REVIEW

File: grant/grant_test.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ func TestGrants(t *testing.T) {
 	assert.NoError(t, err)
 
 	testPGP := secrets.OpenPGPSecret{
-		ID:   "10449759736975846181",
-		Data: []byte(keyPrivate),
+		PrivateID: "10449759736975846181",
+		Data:      []byte(keyPrivate),
```
Comment (gregdhill) [inline grant/grant_test.go]: `keyPrivate` is already a byte slice

File: grant/openpgp.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ func OpenPGPGrant(ref *reference.Ref, public string, private *secrets.OpenPGPSec
 		return nil, err
 	}
 
-	id, err := strconv.ParseUint(private.ID, 10, 64)
+	id, err := strconv.ParseUint(private.PrivateID, 10, 64)
```
Comment (gregdhill) [inline grant/openpgp.go]: Can we rename `private` to `key` or something?

File: grant/grant_test.go (hunk 1 above)
Comment (silasdavis) [reply grant/grant_test.go]: I did only change the formatting though...

File: grant/openpgp.go (hunk 2 above)
Comment (silasdavis) [reply grant/openpgp.go]: yes!

**Comment by gregdhill** [comment]
Refines #35

**Pull request merged**